cpp/src/io/utilities/file_io_utilities.cpp

```diff
@@ -75,8 +75,8 @@ class cufile_shim {
 
   ~cufile_shim()
   {
-    if (driver_close) driver_close();
-    if (cf_lib) dlclose(cf_lib);
+    if (driver_close != nullptr) driver_close();
+    if (cf_lib != nullptr) dlclose(cf_lib);
```

Comment:
why check `!=nullptr`?
`if(ptr)` invokes same result as `if(ptr!=nullptr)` right?

Reply:
That's correct, this is just a minor style change. Avoiding the implicit cast to bool and making it more obvious what we're checking here (type of `cf_lib` is not apparent in the old version).